api/src/main/java/com/vimeo/networking2/requests/ApiResponse.kt

```diff
@@ -0,0 +1,40 @@
+package com.vimeo.networking2.requests
+
+import com.vimeo.networking2.ApiError
+import com.vimeo.networking2.requests.ApiResponse.Failure
+import com.vimeo.networking2.requests.ApiResponse.Success
+
+/**
+ * Result of the API response. [Success] contains the accessToken while the
+ * [Failure] class will inform you of an api error.
+ */
+sealed class ApiResponse<out T> {
+
+    /**
+     * A successful response.
+     *
+     * @param accessToken The accessToken parsed by this successful response.
+     */
+    data class Success<out T>(val accessToken: T) : ApiResponse<T>()
```

Comment:
this looks like a generic response enumeration, but the property is called `accessToken`, should it be called `data` or something similar?

Reply:
bump